extensions/amp-video/0.1/amp-video.js

```diff
@@ -548,6 +549,13 @@ export class AmpVideo extends AMP.BaseElement {
           }
           this.video_.appendChild(currSource);
         });
+
+        return;
```

Comment:
Would this return do anything? Because it's inside the `(source) => {}` function, it doesn't stop the other code from executing

Reply:
Not really it's just an additional safeguard so it doesn't try to run the other condition below. This wouldn't have any impact but 🤷

Reply:
Could it be easier to just do an `else if` then?

Reply:
https://twitter.com/gmajoulet/status/1416089441169444866 🤣